### PR TITLE
Update and extend experimental create_policy tool

### PR DESCRIPTION
**Pull request by stefanberger**
This PR updates and extends the experimental create_policy tool that one can use to add keys or hashes to a Keylime runtime policy.

### CODE REVIEW

**Comment by THS-on** [comment]
The changes LGTM. Do we want to make this tool an official part of the Keylime commandline tools?

**Review by THS-on — APPROVED** [review]

File: scripts/create_policy [hunk 1]
```diff
@@ -129,15 +131,46 @@ def process_ima_buf_in_measurement_list(
...
+        policy["verification-keys"] = verification_key_list
+
+    return policy
+
+
 def main(argv):
     """main"""
-    parser = argparse.ArgumentParser(argv[0])
+    parser = argparse.ArgumentParser(
+        prog=argv[0], description="This is an experimental tool for adding items to a Keylime's IMA runtime policy"
```
Comment (aplanas) [inline scripts/create_policy:166]: `prog=argv[0]` can be dropped, it is taken by default

File: scripts/create_policy [hunk 2]
```diff
@@ -177,18 +210,20 @@ def main(argv):
...
+        "-A",
+        "--add-ima-signature-verification-key",
+        action="append",
+        dest="ima_signature_keys",
+        default=[],
+        help="Add the given IMA signature verification key to the Keylime-internal 'tenant_keyring'; "
+        "the key should be an x509 certificate in DER or PEM format but may also be a public or private key file; "
+        "this option may be passed multiple times",
+    )
     args = parser.parse_args(argv[1:])
```
Comment (aplanas) [inline scripts/create_policy:223]: The parameter `argv[1:]` can also be dropped

File: scripts/create_policy [hunk 3]
```diff
@@ -129,15 +131,46 @@ def process_ima_buf_in_measurement_list(
     return keyrings_map, ima_buf_map, ret
 
 
+def process_signature_verification_keys(verification_keys, policy):
+    """Add the given keys (x509 certificates) to keyring"""
+
+    verification_key_list = None
+    if verification_keys:
+        if policy.get("verification-keys", None):
```
Comment (aplanas) [inline scripts/create_policy:139]: If `policy` is a `dict`, `None` can be elided

**Review by aplanas — APPROVED** [review]

**Comment by mpeters** [comment]
> The changes LGTM. Do we want to make this tool an official part of the Keylime commandline tools?

I think it's fine for it to still be experimental and not installed as part of keylime. Do we reference it in documentation anywhere?

File: scripts/create_policy (hunk 3 above)
Comment (stefanberger) [reply scripts/create_policy]: Fixed.

File: scripts/create_policy (hunk 1 above)
Comment (stefanberger) [reply scripts/create_policy]: Thanks. Fixed.

**Comment by THS-on** [comment]
> Do we reference it in documentation anywhere?

@mpeters I don't think so, but I'm also currently rewriting the section on IMA because it is now out of date.

**Comment by THS-on** [comment]
Failure has nothing to do with change, so I'll merge it.

**Pull request merged**